packages/react-sdk/src/components/Whiteboard/SvgCanvas/SvgCanvas.tsx

```diff
@@ -176,9 +242,12 @@ export const SvgCanvas = forwardRef(function SvgCanvas(
                   }),
             }}
             viewBox={`0 0 ${viewportWidth} ${viewportHeight}`}
+            onClick={handleCanvasClick}
             onMouseDown={onMouseDown}
             onMouseMove={handleMouseMove}
-            onWheel={onWheel}
+            onKeyDown={handleKeyDown}
+            onKeyUp={handleKeyUp}
+            tabIndex={0}
```

Comment:
would be nice if user could use up and down arrows already when he opens standalone, but not sure if that should be part of this PR